precommit/src/main/shell/test-patch.d/github.sh

```diff
@@ -610,9 +610,11 @@ function github_finalreport
     comment=$(echo "${ourstring}"  | cut -f5 -d\|)
 
     if [[ "${vote}" = "H" ]]; then
-      echo "||| _${comment}_ |" >> "${commentfile}"
+      echo "|||| _${comment}_ |" >> "${commentfile}"
+    elif [[ "${vote}" = "-1" ]]; then
+      echo "| :broken_heart: ${TP_VOTE_TABLE[${i}]}" >> "${commentfile}"
     else
-      echo "${TP_VOTE_TABLE[${i}]}" >> "${commentfile}"
+      echo "| :+1: ${TP_VOTE_TABLE[${i}]}" >> "${commentfile}"
```

Comment:
There are three other values that need handling here, +0, -0, and 0.  Marking them with the same emoji as +1 is very misleading.

Reply:
If we use different emoji for these vote. I thought we should use emoji + text vote both.